src/Backoffice/Employee/Domain/DataTransferObject/EmployeDto.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Employee\Domain\DataTransferObjects;
+
+class EmployeeDto
+{
+    public function __construct(
+        private readonly string $name,
+        private readonly string $email,
+    ) {
+    }
+
+    public function getName(): string
+    {
+        return $this->name;
+    }
+
+    public function getEmail(): string
+    {
+        return $this->email;
+    }
+}
```

Comment:
You can avoid defining the getters via `public`. It is important to know that we still keep the encapsulation of the class due to the `readonly` trait. Then, you can access the attributes via `employee->name` and `employee->email`

```suggestion
class EmployeeDto
{
    public function __construct(
        public readonly string $name,
        public readonly string $email,
    ) {
    }
}
```

Reply:
Also you can make the class readonly and dont need to specify it on each attribute!